client/flutter/lib/components/themed_text.dart

```diff
@@ -117,7 +117,7 @@ class ThemedText extends StatelessWidget {
           fontWeight: FontWeight.normal,
           fontSize: 16,
           height: 1.375,
-          letterSpacing: -0.32,
+          letterSpacing: -0.02,
```

Comment:
Double-check this since I'm not 100%, but... I believe that the unit in Figma is `-0.02em`, which is equivalent to `-0.32px` when the font size is 16. The unit of `letterSpacing` is logical pixels [per the docs](https://api.flutter.dev/flutter/painting/TextStyle/letterSpacing.html). I think `-0.32` is right

Reply:
You're right - reverted